cmd/node_agent/na/gcp.go

```diff
@@ -72,3 +72,7 @@ func (na *gcpPlatformImpl) GetServiceIdentity() (string, error) {
 	// TODO(wattli): update this once we are ready for GCE
 	return "", nil
 }
+
+func (na *gcpPlatformImpl) GetAgentCredential() ([]byte, error) {
+	return []byte{}, nil
```

Comment:
Return the JWT or return "not implemented" error?

Reply:
On GCP, is node agent credential needed? I thought it's done by setting the gRPC dial option. I'll change it to return JWT.